<!doctype html>
<html>
  <head>
    <title>LGSF Dashboard</title>
    <link rel="stylesheet"  href="/lgsf-dashboard/assets/css/styles.css">
    <link rel="shortcut icon" href="https://democracyclub.org.uk/static/images/logo_icon.png">
  </head>
  <body>
    <div class="ds-page">
      <a class="ds-skip-link" href="#main">skip to content</a>

      <header class="ds-header">
        <a class="ds-skip-link" href="#main">skip to content</a>
        <a class="ds-logo" href="/lgsf-dashboard/">
          <img src="https://DemocracyClub.github.io/design-system/images/logo_icon.svg" alt="" />
          <span>LGSF Logbooks</span>
        </a>

      </header>


      <main id="main" tabindex="-1" class="ds-stack">

        
<style>

    pre {
        height:400px;
        overflow-x: scroll;
        width:100%
    }
</style>




  


  <h2 id="2023-09-15-14-45">2023-09-15</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>7 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-15 14:45:17.186577</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-15 14:45:24.871756</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:45:17] Fetching Scraper for: HUN                              handlers.py:23
[14:45:21] Begin attempting to scrape: HUN                        handlers.py:27
[14:45:23] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:45:24] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-14-14-02">2023-09-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-14 14:02:01.927773</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-14 14:02:03.991697</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:02:01] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[14:02:02] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:02:03] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-13-12-24">2023-09-13</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-13 12:24:07.112968</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-13 12:24:09.311296</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:24:07] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:24:08] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:24:09] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-12-12-22">2023-09-12</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-12 12:22:35.939019</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-12 12:22:38.234149</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:22:35] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[12:22:36] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:22:37] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:22:38] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-11-14-23">2023-09-11</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-11 14:23:16.385736</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-11 14:23:18.560886</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:23:16] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:23:17] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:23:18] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-10-12-52">2023-09-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-10 12:52:29.333641</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-10 12:52:31.485201</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:52:29] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[12:52:30] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:52:31] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-09-14-08">2023-09-09</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-09 14:08:31.701089</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-09 14:08:33.853282</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:08:31] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[14:08:32] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:08:33] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-08-12-15">2023-09-08</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-08 12:15:59.133630</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-08 12:16:01.359838</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:15:59] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:16:00] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[12:16:01] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-07-13-01">2023-09-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-07 13:01:42.682262</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-07 13:01:44.732257</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:01:42] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[13:01:43] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:01:44] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-06-14-27">2023-09-06</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-06 14:27:41.594381</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-06 14:27:43.712090</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:27:41] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:27:42] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:27:43] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-05-13-40">2023-09-05</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-05 13:40:04.393320</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-05 13:40:08.522498</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:40:04] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:40:05] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:40:06] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[13:40:08] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-09-04-12-54">2023-09-04</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>10 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-09-04 12:54:19.988341</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-09-04 12:54:30.782075</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[12:54:19] Fetching Scraper for: HUN                              handlers.py:23
[12:54:20] Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[12:54:21] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[12:54:30] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-08-28-14-16">2023-08-28</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>25 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-28 14:16:39.773306</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-28 14:17:04.962543</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:16:39] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
[14:16:40] Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:16:41] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[14:17:04] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-08-21-13-40">2023-08-21</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>4 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-21 13:40:09.137543</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-21 13:40:13.782201</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:40:09] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:40:10] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:40:11] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[13:40:13] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-08-14-14-51">2023-08-14</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>47 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-14 14:51:42.282135</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-14 14:52:29.677588</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:51:42] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[14:51:43] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
           ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[14:52:29] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-08-07-13-44">2023-08-07</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-08-07 13:44:15.639623</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-08-07 13:44:17.944490</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:44:15] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[13:44:16] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:44:17] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-07-31-15-01">2023-07-31</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-31 15:01:36.494181</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-31 15:01:38.956599</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[15:01:36] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[15:01:37] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[15:01:38] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-07-24-14-35">2023-07-24</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>3 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-24 14:35:45.516679</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-24 14:35:49.176186</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:35:45] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:35:46] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:35:47] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
[14:35:49] 500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-07-17-14-26">2023-07-17</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-17 14:26:50.272722</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-17 14:26:52.379103</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[14:26:50] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
[14:26:51] Getting all files in Councillors...                       base.py:191
           ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[14:26:52] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  

  


  <h2 id="2023-07-10-13-58">2023-07-10</h2>
  <div class="ds-card">
  <div class="ds-card-body">
    <dl class="ds-descriptions">
      <div>
        <dt>Duration</dt>
        <dd>2 seconds</dd>
      </div>
      <div>
        <dt>Start</dt>
        <dd>2023-07-10 13:58:01.118617</dd>
      </div>
      <div>
        <dt>End</dt>
        <dd>2023-07-10 13:58:03.482425</dd>
      </div>
      <div>
        <dt>Status code</dt>
        <dd>1</dd>
      </div>
      <div>
        <dt>Error</dt>
        <dd>Traceback (most recent call last):
  File "/var/task/lgsf/aws_lambda/handlers.py", line 32, in scraper_worker_handler
    scraper.run(run_log)
  File "/var/task/lgsf/councillors/scrapers.py", line 179, in run
    wards = self.get_councillors()
  File "/var/task/lgsf/councillors/scrapers.py", line 196, in get_councillors
    req = self.get(self.format_councillor_api_url(), verify=self.verify_requests)
  File "/var/task/lgsf/scrapers/base.py", line 49, in get
    response.raise_for_status()
  File "/opt/python/requests/models.py", line 1021, in raise_for_status
    raise HTTPError(http_error_msg, response=self)
requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: http://applications.huntingdonshire.gov.uk/moderngov//mgWebService.asmx/GetCouncillorsByWard
</dd>
      </div>
    </dl>
  </div>
  </div>

  <h3>Run log</h3>
  <pre>[13:58:01] Fetching Scraper for: HUN                              handlers.py:23
           Begin attempting to scrape: HUN                        handlers.py:27
           Deleting existing data...                                 base.py:239
           Getting all files in Councillors...                       base.py:191
[13:58:02] ...found 1 files in Councillors                           base.py:207
           Deleting batch no. 1 consisting of 1 files                base.py:216
[13:58:03] ...data deleted.                                          base.py:246
           Scraping from                                              base.py:42
           http://applications.huntingdonshire.gov.uk/moderngov//mgWe           
           bService.asmx/GetCouncillorsByWard                                   
           500 Server Error: Internal Server Error for url:       handlers.py:36
           http://applications.huntingdonshire.gov.uk/moderngov//               
           mgWebService.asmx/GetCouncillorsByWard                               
           Finished attempting to scrape: HUN                        base.py:324
</pre>
  


      </main>
      <footer class="ds-footer">
        <div class="ds-block-centered ds-text-centered ds-stack">
          <div class="ds-cluster-center">
            <ul>
              <li><a href="/lgsf-dashboard/api/failing.json">Failing JSON</a></li>
            </ul>
          </div>
          <div class="ds-copyright">
            <a href="https://democracyclub.org.uk/">
              <img src="https://DemocracyClub.github.io/design-system/images/logo_white_text.svg" alt="Democracy Club Home" />
            </a>
            <p>Copyright © 2021 Democracy Club</p>
            <p>Community Interest Company</p>
            <p>Company No: <a href="https://beta.companieshouse.gov.uk/company/09461226">09461226</a></p>
          </div>
        </div>
      </footer>
    </div>
  </body>
